diagnostic.md

```diff
@@ -34,5 +45,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each model as |phone|}}
+      {{my-contact/my-phone phone=phone}}
+    {{/each}}
```

Comment:
So close! Because this is a component, it's data is not defined as model.  Although I suppose it could be, so I'll give it to you.
`{{#each contact.phones as |phone|}} `
👍